gpii/node_modules/flatMatchMaker/test/data/pre_mm_payload2.json

```diff
@@ -53,7 +53,7 @@
                         "HighContrastOn": {
                             "schema": {
                                 "title": "High Contrast",
-                                "description": "Whether to enable/disable High Contrast",
+                                "description": "Whether to enable/disable High Contrast.",
```

Comment:
I wonder why we acquired full stops in these descriptions - it is odd since this doesn't agree with the text in the SR - does this get verified by the tests?

Reply:
I suspect we are testing this canned solutions data off to the side with canned expected output.  IMO part of the SR refactoring should involve eliminating as much of this as possible, and only having non-SR solutions test data for things like testing error handling.